### PR TITLE
Check source repository in Travis

### PR DESCRIPTION
**Pull request by astj**
Checks whether repository is accessible by `curl`.

### CODE REVIEW

File: t/plugins.t [hunk 1]
```diff
@@ -16,6 +16,15 @@ for my $file (<./plugins/*.json>) {
         ok !$@, "valid JSON";
         ok $json->{source}, "has source";
         ok $json->{description}, "has description";
+
+        my $valid = ok $json->{source} =~ m|\A[-_.a-zA-Z0-9]+/[-_.a-zA-Z0-9]+\Z|, 'source is valid';
```
Comment (karupanerura) [inline t/plugins.t]: Should use https://metacpan.org/pod/Test::More#like, it makes more helpful error message.

Comment (astj) [reply t/plugins.t]: Thank you! fixed at https://github.com/mackerelio/plugin-registry/pull/10/commits/981a6efd4aaced03ae5fa2976915a497b236fecc

File: t/plugins.t [hunk 2]
```diff
@@ -16,6 +16,15 @@ for my $file (<./plugins/*.json>) {
         ok !$@, "valid JSON";
         ok $json->{source}, "has source";
         ok $json->{description}, "has description";
+
+        my $valid = like $json->{source}, qr|\A[-_.a-zA-Z0-9]+/[-_.a-zA-Z0-9]+\Z|, 'source is valid';
+        # test only safe sources
+        if ($valid) {
+            # curl -f exists with 0 when request succeeded
```
Comment (itchyny) [inline t/plugins.t]: Probably a typo of `exits`.

Comment (astj) [reply t/plugins.t]: Oh…
https://github.com/mackerelio/plugin-registry/pull/10/commits/740a0c7490b4596ffdb95150167637f4ee74011b

**Comment by itchyny** [comment]
LGTM

**Pull request merged**